users.json

```diff
@@ -5,198 +5,209 @@
       {
         "name": "Alejandro Escamilla",
         "email": "alejandro@reboot.com",
-        "password": 123456,
+        "role": "partner",
+        "password": "123456",
         "img": "https://randomuser.me/api/portraits/men/1.jpg",
-        "location": "Las Palmas"
+        "location": "5ea70cb5a16c41205f7b7918"
```

Comment:
no puedes meter in ObjectID aqui en literal, porque cambia cada vez que llamas al seed. Si quieres lo puedes meter en el seed.js llamando directamente a los modelos de mongoose y haciendo un `Location.find`, guardas el ID y luego `Users.Update()`